packages/angular/src/generators/library/lib/update-project.ts

```diff
@@ -93,12 +94,17 @@ function updateFiles(host: Tree, options: NormalizedSchema) {
       });
     });
           `
-    );
+      );
+    }
+  } else {
+    host.delete(joinPathFragments(libRoot, `${options.fileName}.module.ts`));
   }
 
   host.write(
     `${options.projectRoot}/src/index.ts`,
-    `
+    options.skipFormat
```

Comment:
I guess this should be `options.skipModule`, but we could also just add this `host.write()` call to the `if` block above.

Reply:
I'm going to keep it this way, as I would still want `index.ts` file to be created, and I'm not sure it would be if it was in the if block, but I'll fix the option. I made that mistake many times yesterday, i must have missed this one 🤦